diff -Nur HappyDoc-r2_0_1.orig/happydoclib/docset/docset_mstruct.py HappyDoc-r2_0_1/happydoclib/docset/docset_mstruct.py
--- HappyDoc-r2_0_1.orig/happydoclib/docset/docset_mstruct.py	Wed Oct 24 23:27:35 2001
+++ HappyDoc-r2_0_1/happydoclib/docset/docset_mstruct.py	Thu Feb  7 02:31:36 2002
@@ -39,9 +39,9 @@
     'created':'24.08.2001',
 
     # current information
-    'author':'$Author$',
-    'version':'$Revision$',
-    'date':'$Date$',
+    'author':'$Author$',
+    'version':'$Revision$',
+    'date':'$Date$',
     'locker':'$Locker$',
 }
 
@@ -72,7 +72,7 @@
     """
 
     def __init__(self, **args):
-        happydocset.DocSet.__init__(self, **args)
+        happydoclib.happydocset.DocSet.__init__(self, **args)
 
 
     def openOutput(self, name, title, subtitle, rtype=INDEX_FILE):
@@ -172,7 +172,7 @@
         formatter.moduleIndex(module, self._index_file)
         
         output_name=self._formatter.getFullOutputNameForObject(module)
-        output=self.openOutput(output_name, 'Module: %s' % module_name, module.getFileName(), MODULE_FILE)
+        output=self.openOutput(output_name, 'Module: %s' % module_name, module.getFilename(), MODULE_FILE)
 
         # start the module tag
         formatter.processModule(module, output, START)
@@ -205,7 +205,7 @@
                 formatter.classIndex(c, output)
                 class_output_name=formatter.getFullOutputNameForObject(c)
                 class_output=self.openOutput(class_output_name, 'Class: %s' % class_name, \
-                    module.getFileName(), CLASS_FILE)
+                    module.getFilename(), CLASS_FILE)
                 self._writeClass(module, class_name, class_output)
                 self.closeOutput(class_output, CLASS_FILE)
             formatter.processClass(None, output, POST)
diff -Nur HappyDoc-r2_0_1.orig/happydoclib/formatter/formatter_DocBookX.py HappyDoc-r2_0_1/happydoclib/formatter/formatter_DocBookX.py
--- HappyDoc-r2_0_1.orig/happydoclib/formatter/formatter_DocBookX.py	Sun Feb  3 22:50:45 2002
+++ HappyDoc-r2_0_1/happydoclib/formatter/formatter_DocBookX.py	Thu Feb  7 02:31:07 2002
@@ -22,7 +22,7 @@
 
    Example:
 
-       happydoc -T mstruct -F xmldocbook my_module formatter_encoding=utf-8
+       happydoc -T mstruct -F docbookx my_module formatter_encoding=utf-8
 
        xsltproc --xinclude my_custom_docbook_to_html.xsl my_module/index.docb > my_module.html
 """
@@ -45,9 +45,9 @@
     'created':     '24-08-2001',
 
     # current information
-    'author':      '$Author$',
-    'version':     '$Revision$',
-    'date':        '$Date$',
+    'author':      '$Author$',
+    'version':     '$Revision$',
+    'date':        '$Date$',
 }
 
 
@@ -128,8 +128,8 @@
         """
         if stage==START:
             self.tag('section', output, {'xmlns:xi': 'http://www.w3.org/2001/XInclude'})
-            name=info.getFullyQualifiedName(self.getFilenamePrefix())
-            name=string.replace(name, '/', '.')[:len(name)-3]
+            name=info.getFullyQualifiedName()
+            name=string.replace(name, '/', '.')[:-3]
             self.writeTaggedText('title', 'Module %s' % name, output)
         elif stage==END:
             self.endTag('section', output)
@@ -270,7 +270,7 @@
         if package_info.getName():
             pname=package_info.getName()
         else:
-            pname=package_info.getFileName()
+            pname=package_info.getFilename()
 
         name='%s/%s#%s' \
             % (pname, self.getIndexFileName(), 'xpointer(/chapter/section)')
diff -Nur HappyDoc-r2_0_1.orig/happydoclib/formatter/xmlformatterbase.py HappyDoc-r2_0_1/happydoclib/formatter/xmlformatterbase.py
--- HappyDoc-r2_0_1.orig/happydoclib/formatter/xmlformatterbase.py	Sun Feb  3 22:50:11 2002
+++ HappyDoc-r2_0_1/happydoclib/formatter/xmlformatterbase.py	Thu Feb  7 02:31:15 2002
@@ -26,9 +26,9 @@
     'created':     '24-08-2001',
 
     # current information
-    'author':      '$Author$',
-    'version':     '$Revision$',
-    'date':        '$Date$',
+    'author':      '$Author$',
+    'version':     '$Revision$',
+    'date':        '$Date$',
 }
 
 
@@ -204,7 +204,7 @@
         return string.join(map(lambda attr_item: '%s="%s"' % attr_item, attrs.items()))
 
 
-    def writeText(self, text, output, textFormat, quote=1):
+    def writeText(self, text, output, textFormat=None, quote=1):
         """Outputs text to XML file.
 
         Arguments
